### PR TITLE
POL-700 Clean up post-migration dead code

### PR DESCRIPTION
**Pull request by gwenneg**
image

### CODE REVIEW

**Comment by gwenneg** [comment]
This is ready for a review :white_check_mark:

File: src/main/resources/db/migration/V22__POL-700_org_id_migration_cleanup.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE account_latest_update;
+
+DROP RULE rule_policies_history_last_triggered ON policies_history;
+
+CREATE RULE rule_policies_history_last_triggered AS ON INSERT TO policies_history
+DO UPDATE policy
+   SET last_triggered = GREATEST(last_triggered, NEW.ctime)
+   WHERE id = uuid(NEW.policy_id) and org_id = NEW.org_id;
```
Comment (gwenneg) [inline src/main/resources/db/migration/V22__POL-700_org_id_migration_cleanup.sql:8]: This needs to be merged ASAP. I might move that to another PR.

File: src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java [hunk 2]
```diff
@@ -173,36 +172,4 @@ private void putToBucket(Map<String, Integer> buckets, int count) {
         buckets.computeIfPresent(bucket, (b, c) -> buckets.get(b) + 1);
         buckets.putIfAbsent(bucket, 1);
     }
-
-    // Trigger Tenant Tuple
-    public static class TTT {
```
Comment (josejulio) [inline src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java:178]: Not used anymore? or is there an OrgId replacement?

**Review by josejulio — APPROVED** [review]
LGTM - minor comment

File: src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java (hunk 2 above)
Comment (gwenneg) [reply src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java:178]: I think it stopped being used when policies was refactored. No link with the org ID migration.

**Pull request merged**